SPEC.md

```diff
@@ -10164,6 +10112,78 @@ Example output:
 </p>
 </details>
 
+### `length`
+
+```
+Int length(Array[X]|Map[X, Y]|Object|String)
+```
+
+Returns the length of the input argument as an `Int`:
+
+* For an `Array[X]` argument: the number of elements in the array.
+* For a `Map[X, Y]` argument: the number of items in the map.
+* For an `Object` argument: the number of key-value pairs in the object.
+* For a `String` argument: the number of characters in the string.
+
+Note that every struct of a given type will always have the same length, equal to the number of members in the struct definition, regardless of whether any of the members are optional.
+
```

Comment:
Since struct's are not allowed as an argument, this comment seems unnecessary.

Reply:
good catch, thanks